test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -166,9 +168,11 @@ public static void ClearProfilerEnvironmentVariables()
                 environmentVariables["COR_ENABLE_PROFILING"] = "1";
                 environmentVariables["COR_PROFILER"] = EnvironmentHelper.ProfilerClsId;
                 environmentVariables["COR_PROFILER_PATH"] = GetProfilerPath();
+
+                processName = Path.GetFileName(processPath);
             }
 
-            environmentVariables["DD_PROFILER_PROCESSES"] = processPath;
+            environmentVariables["DD_PROFILER_PROCESSES"] = processName;
```

Comment:
We can probably drop `DD_PROFILER_PROCESSES` entirely for these tests. It was really meant to be used when running Visual Studio using `devenv.bat`.

Reply:
Do users also utilize this variable?

Reply:
Removing this makes the tests which rely on IISExpress fail actually.